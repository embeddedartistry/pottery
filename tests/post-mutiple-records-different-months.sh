pottery init
TEST_TIME=1992-01-12T16:34:23Z pottery post "Honey pie, I don't know when it started, stop buying your albums from the supermarket
They only sell records that have charted, and Art Brut, we've only just started"
TEST_TIME=1992-03-01T10:15:01Z pottery post "And yes, this is my singing voice, it's not irony, it's not rock and roll
We're just talking, to the kids"
ls -R docs/project-history
cat docs/project-history/1992/1992-01/1992-01-12T16:34:23Z.md
cat docs/project-history/1992/1992-03/1992-03-01T10:15:01Z.md
