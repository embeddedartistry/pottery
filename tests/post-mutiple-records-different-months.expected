pottery init
TEST_TIME=1992-01-12T16:34:23Z pottery post "Honey pie, I don't know when it started, stop buying your albums from the supermarket
They only sell records that have charted, and Art Brut, we've only just started"
TEST_TIME=1992-03-01T10:15:01Z pottery post "And yes, this is my singing voice, it's not irony, it's not rock and roll
We're just talking, to the kids"
ls -R docs/project-history
1992

docs/project-history/1992:
1992-01
1992-03

docs/project-history/1992/1992-01:
1992-01-12T16:34:23Z.md

docs/project-history/1992/1992-03:
1992-03-01T10:15:01Z.md
cat docs/project-history/1992/1992-01/1992-01-12T16:34:23Z.md
Time: 1992-01-01 16:34

Honey pie, I don't know when it started, stop buying your albums from the supermarket They only sell records that have charted, and Art Brut, we've only just started
cat docs/project-history/1992/1992-03/1992-03-01T10:15:01Z.md
Time: 1992-03-03 10:15

And yes, this is my singing voice, it's not irony, it's not rock and roll We're just talking, to the kids
